lang/en_ar/format_topcoll.php

```diff
@@ -224,3 +224,7 @@
 $string['defaulttoggleborderradiusbr_desc'] = 'Border bottom right radius of thy toggle.';
 $string['defaulttoggleborderradiusbl'] = 'Toggle bottom left border radius';
 $string['defaulttoggleborderradiusbl_desc'] = 'Border bottom left radius of thy toggle.';
+
+// Toggle Display Blocks
```

Comment:
Not needed as no change from en version and thus Moodle uses that instead if no entry here.